application/event_processor/event_processor.py

```diff
@@ -118,7 +118,7 @@ def send_changes(self, message_received: int, record_id: str, sequence_number: i
             logger.debug(
                 "CR to send",
                 extra={
-                    "change_request": change_request,
+                    "change_request": change_payload,
```

Comment:
Suggest changing the key `change_request` text to `change_payload` or `change_request_payload` to better reflect what it holds.